.env.example

```diff
@@ -5,9 +5,11 @@ QUML_ODK_QUESTION_BANK_DETAILS_URL='https://dock.sunbirded.org/api/question/v1/l
 
 # Vairables for form upload
 UPLOAD_FORMS=FALSE # FALSE = Disabled, TRUE = Enabled
+UPLOAD_FORM_TO=   # AGGREGATE (or) CENTRAL
```

Comment:
Change this to ODK_BACKEND_TYPE

Reply:
Fixed. Refereance commit: https://github.com/Samagra-Development/quml2xform/pull/7/commits/1adf1871a6f5784955944572af0d08e6f23825c1